CHANGELOG.md

```diff
@@ -4,6 +4,25 @@
 
 - "You miss 100 percent of the chances you don't take. — Wayne Gretzky" — Michael Scott
 
+## 7.18.0
+
+This release adds the `beforeSendTransaction` callback to all JS SDKs, letting you make changes or drop transactions before they're sent to Sentry. This callback works identically to `beforeSend`, just for transactions.
+
+Furthermore, this release adds the `shouldCreateSpanForRequest` option to the Node SDK and the `tracePropagationTargets` option to the Browser SDK, providing fine grained control over which outgoing Http requests should be represented as a span in an ongoing transaction and which requests should be traced by Sentry.
```

Comment:
H: I think we should hold off on talking about `shouldCreateSpanForRequest`, too, because I realized it (and the node `tracePropagationTargets` option before it) got added in the wrong spot. 😬

Reply:
Hmm I also realized that the work around these options [isn't completed yet](https://github.com/getsentry/sentry-javascript/issues/5285#issuecomment-1290503283). I'll remove the note.

Reply:
[f6f1ec5](https://github.com/getsentry/sentry-javascript/pull/6151/commits/f6f1ec59879b4e472b4ee5c664be7a11fec08de0)